physics/continuous_trajectory.hpp

```diff
@@ -132,27 +132,46 @@ class ContinuousTrajectory : public Trajectory<Frame> {
   ContinuousTrajectory();
 
  private:
+  // Each polynomial is valid over an interval [t_min, t_max].  Polynomials are
+  // stored in this vector sorted by their |t_max|, as it turns out that we
+  // never need to extract their |t_min|.  Logically, the |t_min| for a
+  // polynomial is the |t_max| of the previous one.  The first polynomial has a
+  // |t_min| which is |*first_time_|.
+  struct InstantPolynomialPair {
+    InstantPolynomialPair(
+        Instant t_max,
+        not_null<std::unique_ptr<Polynomial<Displacement<Frame>, Instant>>>
```

Comment:
Why are they not Position-valued?

Reply:
Mostly because they were not in Chebyshev, and I wanted to be able to compare apples to apples.  We could look into this in the future (it would probably be less clunky than with Chebyshev because the metaprogramming would just do the right thing).  Nod to #770.